CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.17.0] - 2022-07-20
```

Comment:
It seems that we messed up the changelog. The changes for the allowlist should show up here, not in 0.16.0

Reply:
Must have been messed up when merging with main. I'll fix it with a push to main